docs/components/InputDate/InputDate.stories.mdx

```diff
@@ -89,7 +68,7 @@ group.
 
 ## Related components
```

Comment:
Should we follow the order of sections as laid out in the ticket?  Accessibility would come last...
```
Design & usage guidelines

Content guidelines

Related components

Related patterns

Related resources

Platform considerations

Accessibility
```